python_modules/libraries/dagster-github/dagster_github/resources.py

```diff
@@ -1,118 +1,159 @@
 import time
 from datetime import datetime
+from typing import Any, Dict, Optional
 
 import jwt
 import requests
-from dagster import Field, IntSource, StringSource, resource
+from dagster import ConfigurableResource, resource
+from dagster._config.structured_config import infer_schema_from_config_class
+from dagster._utils.cached_method import cached_method
+from pydantic import Field as PyField
```

Comment:
Are we aligning on doing `Field as PyField` to disambiguate from `dagster.Field`? I've just been importing as `Field` since `dagster.Field` doesn't get imported anymore

Reply:
I think importing `pydantic.Field` as just `Field` is correct - `PyField` is a bad habit of mine from context where they've coexisted (e.g. working on the internals of Pythonic config/resources). Will change!